apps/rush-lib/src/logic/Git.ts

```diff
@@ -207,6 +207,17 @@ export class Git {
     return result;
   }
 
+  public getFileContent(targetBranch: string, gitRelativePath: string, repositoryRoot: string): string {
```

Comment:
```suggestion
  public getFileContentAtRef(targetRef: string, gitRelativePath: string, repositoryRoot: string): string {
```

Reply:
Might also make sense to pass an options object to this function to prevent someone from messing up the parameter order.